pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/forward/BaseChunkForwardIndexReader.java

```diff
@@ -446,9 +446,10 @@ public void readValuesSV(int[] docIds, int length, double[] values, ChunkReaderC
   }
 
   @Override
-  public void close() {
+  public void close() throws IOException {
```

Comment:
(code format) This doesn't follow `Pinot Style`, and there are some other places with the same issue. Can you try to auto reformat all the changes?

Reply:
hmm, that is very extrange, I am not sure what happened. I always run `mvn checkstyle:check` before pushing a commit. In fact I forgot to do this and had to push another commit. The ^ ^ command succeeded!! 
So I have opened all the files I touched and done a cmd+alt+shift+l to reformat according to `config/codestyle-intellij.xml`. A lot more lines than I expected were formatted :D .

sorted in d02d798a0879323bf2cbaccf225d617087305532